doc/source/examples/permutation_importance_classification_leave.ipynb

```diff
@@ -0,0 +1,933 @@
+{
```

Comment:
<p>Since <code>seaborn</code> isn't a required dependency, I would add a sentence about requiring it for the example and a cell calling <code>!pip install seaborn</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/798/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>